cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -1769,12 +1771,17 @@ impl_runtime_apis! {
 			build_state::<RuntimeGenesisConfig>(config)
 		}
 
-		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
-			get_preset::<RuntimeGenesisConfig>(id, |_| None)
+		fn get_preset(id: &Option<PresetId>) -> Option<Vec<u8>> {
+			get_preset::<RuntimeGenesisConfig>(id, &genesis_config_presets::get_preset)
 		}
 
-		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
-			vec![]
+		fn preset_names() -> Vec<PresetId> {
+			use crate::genesis_config_presets::preset_names::*;
+			vec![
+				PresetId::from(PRESET_GENESIS),
+				PresetId::from(PRESET_DEVELOPMENT),
+				PresetId::from(PRESET_LOCAL),
```

Comment:
@michalkucharczyk can you please move this vec construction to the `genesis_config_preset.rs`, e.g. as I did here: https://github.com/paritytech/polkadot-sdk/pull/5327/files#diff-9a96444b3aa8ae9eb26c5e7ca353e8225771adf47d599ebc6f0a7cd85343e356R97-R103

Reply:
done in: https://github.com/paritytech/polkadot-sdk/pull/3996/commits/13f411d622f0ca5e0f6e4ad168108ef368205673